minimesos/src/main/java/com/containersol/minimesos/marathon/MarathonContainer.java

```diff
@@ -78,6 +78,19 @@ public void setZooKeeper(ZooKeeper zooKeeper) {
     }
 
     @Override
+    public void deleteApp(String app) {
+        try {
+            HttpResponse<JsonNode> deleteResponse = Unirest.delete(getServiceUrl().toString() + END_POINT_EXT + "/" + app).header(HEADER_ACCEPT, APPLICATION_JSON).asJson();
+            deleteResponse.getBody().getObject();
+            if (!(deleteResponse.getStatus() == HttpStatus.SC_OK)) {
+                throw new MinimesosException("Could not delete app: " + app);
+            }
+        } catch (UnirestException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1166)